week3/74-udalit-kazhdyi-trietii-simvol/task.py

```diff
@@ -0,0 +1,15 @@
+def main():
+    input_file = open("input.txt")
+    s = input_file.read()
```

Comment:
can we not load full file in memory?

Reply:
https://docs.python.org/3/tutorial/inputoutput.html#:~:text=Otherwise%2C%20at%20most%20size%20characters%20(in%20text%20mode)